python/ruff/__main__.py

```diff
@@ -3,6 +3,23 @@
 import sysconfig
 from pathlib import Path
 
+RUFF_PATHS = [
+    Path(sysconfig.get_config_var("userbase")) / "bin" / "ruff",
+    Path(sysconfig.get_path("scripts")) / "ruff",
```

Comment:
We shouldn’t prefer a version from `pip install --user` *over* a currently active virtualenv.